llvm/test/Transforms/GVN/PRE/load-pre-licm.ll

```diff
@@ -282,3 +284,97 @@ header:
   call void @hold(i32 %v1)
   br label %header
 }
+
+
+; In block Z one load is unavailable (via entry block), whereas via other 2
+; predecessors the loads are available.
```

Comment:
As far as GVN knows, the load is not available in block B. If it were, the transform would already be performed.